### PR TITLE
NWD-NWW

### PR DESCRIPTION
**Pull request by TomaszWeglinski2**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: nwd-nnw
🏅 Granted 6 XP for @TomaszWeglinski2

**Comment by ziobron** [comment]
⏱ 3 XP extra for @TomaszWeglinski2 for on-time delivery

File: homework/nwd-nnw/nwdNww.hpp [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
 
 int NWD(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+
+    int c;
+
+
+    while(rhs!=0){
+     c=lhs%rhs;
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:9]: Add space between operations like '+' '-' '!=' '==' etc...

Also what is `c`?  Variable should always represent real operation.
Like `result` `divider` `multiplier` `error_code` etc..

Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:9]: Add space between operations like '+' '-' '!=' '==' etc... and `while` `if` `else` etc...

File: homework/nwd-nnw/nwdNww.hpp [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+     rhs = c;
+
+    }
+    return abs(lhs);
 }
 
 int NWW(int lhs, int rhs) {
-    // TODO: Implement me :)
-    return -1;
+   if (lhs ==0 || rhs ==0){
```
Comment (nauka-programowania-MA) [inline homework/nwd-nnw/nwdNww.hpp:18]: add spaces

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**